apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -22,12 +27,37 @@ h2 + .c-card-wrapper {
     padding: 0;
 }
 
+.c-card-list--fillContainer {
+    grid-template-columns: 1fr;
+
+
+    @include f.media('>=mid') {
+        grid-template-columns: auto auto auto;
+
+        @container (#{$card-container-max-width}) {
+            li:last-child {
+                grid-column: span 2;
+            }
+        }
```

Comment:
I appreciate I'm not bringing an alternative here but I'd be surprised if grid can't do this without container queries

Reply:
That is what I thought, but I've still not found it 😂

Reply:
I think this kind of behaviour is typically best suited to flexbox instead of grid, but it can still be fiddly